coretex/cli/modules/node.py

```diff
@@ -236,6 +236,23 @@ def selectNodeMode(storagePath: str) -> Tuple[int, Optional[int]]:
     return availableNodeModes[selectedMode], None
 
 
+def promptRam(config: Dict[str, Any], ramLimit: int, retryCount: int = 0) -> int:
```

Comment:
`retryCount` is unused except in recursive function calls, it is safe to remove completely.

Reply:
Do we want to keep prompting for ram forever or to end as soon as user enters incorrect value?

Reply:
Forever